concent_api/core/constants.py

```diff
@@ -8,7 +8,7 @@
 SCENE_FILE_EXTENSION = '.blend'
 
 # Defines max length of task_id passed in Golem Messages.
-MESSAGE_TASK_ID_MAX_LENGTH = 128
+MESSAGE_TASK_ID_MAX_LENGTH = 36
```

Comment:
It should be no longer `MAX` but the only allowed length.

Reply:
But it's still `MAX` because if someone delete dashes `-` from ID then it is still available `UUID` ID but string is 4 character shorter.